src/translations/km.json

```diff
@@ -1,67 +1,66 @@
-    "set": "កំណត់",
-    "submitting": "បញ្ជូន",
-    "to": "ទៅ",
-    "tryOpeningAnotherForm": "ព្យាយាមបើកទម្រង់ផ្សេងទៀត",
-    "useValue": "ប្រើតម្លៃ",
-    "weeks": "សប្តាហ៍",
-    "yearsAgo": " ឆ្នាំមុន"
-  }
-  
\ No newline at end of file
+  "aDayAgo": "មួយថ្ងៃមុន",
+  "add": "បន្ថែម",
+  "addHour": "បន្ថែមមួយម៉ោង",
+  "addMinute": "បន្ថែមមួយនាទី",
+  "addSecond": "បន្ថែមមួយវិនាទី",
+  "aFewSecondsAgo": "ប៉ុន្មានវិនាទីមុន",
+  "aMinuteAgo": "មួយនាទីមុន",
+  "aMonthAgo": "មួយ​ខែមុន",
+  "anHourAgo": "មួយ​ម៉ោង​មុន",
+  "aYearAgo": "មួយ​ឆ្នាំមុន",
+  "cancel": "បោះបង់",
+  "changeToMonthView": "ប្តូរទម្រង់ទៅជាខែ",
+  "chooseMonthAndYear": "ជ្រើសរើសខែ និងឆ្នាំ",
+  "clearEntry": "តើអ្នកពិតជាចង់សម្អាតធាតុនេះ?",
+  "closeThisPanel": "បិទប្រអប់នេះ",
+  "componentLoadingFailed": "ការចម្លងទិន្នន័យ​បរាជ័យ...",
+  "daysAgo": " ថ្ងៃ​មុន",
+  "deleteEntry": "តើអ្នកពិតជាចង់លុបធាតុនេះទេ?",
+  "discardButton": "បោះបង់",
+  "enterMoreCharacters": "សូមបញ្ចូលតួអក្សរ ២ ឬច្រើន។",
+  "errorFetchingFormData": "ការទាញយកទិន្នន័យពីទម្រង់មិនដំណើរការ។ ព័ត៌មានលម្អិត៖ {detail}",
+  "errorLoadingForm": "ទម្រង់ចម្លងទិន្នន័យមិនដំណើរការ",
+  "errorWithForm": "ទម្រង់នេះមិនដំណើរការ",
+  "fix": "ជួសជុល",
+  "formSubmissionFailed": "ការបញ្ជូនទម្រង់ទិន្នន័យមិនដំណើរការ។ បញ្ហាមិនដំណើរការ៖ {error}",
+  "formSubmittedSuccessfully": "Tទម្រង់ទិន្នន័យបានបញ្ជូនដោយជោគជ័យ",
+  "from": "ពី",
+  "futureDateRestriction": "មិនអនុញ្ញាតឱ្យមានកាលបរិច្ឆេទនៅថ្ងៃអនាគតទេ",
+  "hoursAgo": " ច្រើន​ម៉ោង​កន្លងទៅ",
+  "invalidDate": "កាលបរិច្ឆេទមិនត្រឹមត្រូវ។ សូមជ្រើសរើសកាលបរិច្ឆេទត្រឹមត្រូវ",
+  "loading": "កំពុងចម្លង...",
+  "loadingComponent": "កំពុង​ចម្លងទិន្នន័យ...",
+  "max": "សូមបញ្ចូលតម្លៃដែលតិចជាងឬស្មើនឹង {max}",
+  "maxDate": "សូមជ្រើសរើសកាលបរិច្ឆេទដែលនៅឬមុន {maxDate}",
+  "maxEntries": "មិនអាចមានច្រើនជាង {max} ធាតុ",
+  "maxLength": "សូមបញ្ចូលតម្លៃដែលមិនលើសពី {maxLength} តួអក្សរ",
+  "min": "សូមបញ្ចូលតម្លៃដែលធំជាងឬស្មើនឹង {min}",
+  "minDate": "សូមជ្រើសរើសកាលបរិច្ឆេទដែលនៅឬក្រោយ {minDate}",
+  "minLength": "សូមបញ្ចូលតម្លៃដែលយ៉ាងហោចណាស់ {minLength} តួអក្សរ",
+  "minusHour": "ដកមួយម៉ោង",
+  "minusMinute": "ដកមួយនាទី",
+  "minusSecond": "ដកមួយវិនាទី",
+  "minutesAgo": " ប៉ុន្មាននាទីមុន",
+  "monthsAgo": " ច្រើន​ខែ​កន្លងមក",
+  "next21Years": "២១ ឆ្នាំក្រោយ",
+  "nextMonth": "ខែក្រោយ",
+  "nextYear": "ឆ្នាំក្រោយ",
+  "patientIdentifierDuplication": "ស្ទួនលេខកូដអត្តសញ្ញាណអ្នកជំងឺ",
+  "patientIdentifierDuplicationDescription": "លេខកូដអត្តសញ្ញាណដែលបានបញ្ចូល សម្រាប់អ្នកជំងឺមានរួចហើយ។ សូមពិនិត្យមើលលេខកូដអត្តសញ្ញាណ ហើយព្យាយាមម្តងទៀត",
+  "previous21Years": "២១ ឆ្នាំមុន",
+  "previousMonth": "ខែមុន",
+  "previousValue": "តម្លៃពីមុន",
+  "previousYear": "ឆ្នាំមុន",
+  "remove": "ដកចេញ",
+  "requiredField": "ប្រអប់​នេះ​ត្រូវតែ​បំពេញ",
+  "saveAndCloseButton": "រក្សាទុក និងបិទ",
+  "selectWeeks": "ជ្រើសរើសសប្តាហ៍",
+  "set": "កំណត់",
+  "submitting": "បញ្ជូន",
+  "to": "ទៅ",
+  "tryOpeningAnotherForm": "ព្យាយាមបើកទម្រង់ផ្សេងទៀត",
+  "useValue": "ប្រើតម្លៃ",
+  "weeks": "សប្តាហ៍",
+  "yearsAgo": " ឆ្នាំមុន"
```

Comment:
I've arranged all these translation keys and strings in ascending order for readability.